common/utils/src/main/resources/error/error-classes.json

```diff
@@ -7902,6 +7902,11 @@
       "Doesn't support month or year interval: <interval>"
     ]
   },
+  "_LEGACY_ERROR_UNKNOWN" : {
```

Comment:
@itholic Why did you name it with the prefix `_LEGACY_`? Do you plan to eliminate it in the future?

Reply:
Because I thought that not having an error class assigned basically meant it was a LEGACY error, but I have not very strong opinion. Do you have any preference? Also cc @srielau FYI

Reply:
> Because I thought that not having an error class assigned basically meant it was a LEGACY error

I would say it is true. `SparkException` can still be raised w/ just a message since it is not fully ported on error classes. For instance:
https://github.com/apache/spark/blob/e170252714e3662c7354321f78a3250114ea7e9e/common/utils/src/main/scala/org/apache/spark/util/SparkThreadUtils.scala#L51-L53

Reply:
Since we know the cases when the error class is not set, how about just name the error class like `UNCLASSIFIED`

Reply:
Sounds reasonable to me. Let me address it.